make intro and code appendix more language agnostic (#505)
worldbank/dime-data-handbook: chapters/0-introduction.tex

which includes instructions for how to write good code,
instructions on how to use the code examples in this book,
as well as our Stata Style Guide.

> DIME projects are strongly encouraged to
> follow coding style guides in their work.
> Style guides harmonize code across teams 
> making it easier to understand and reuse of code,
> which ultimately helps teams to 
> build on each other's best practices.
> Some of the programming languages used at DIME 
> already have well-established and commonly used style guides,
>  for example, the Tidyverse style guide for R
> or PEP-8 for Python.\sidenote{See DIME Analytics Coding Standards:
> 	\url{https://github.com/worldbank/dime-standards}}. 

Reviewer: ```suggestion
DIME projects are strongly encouraged to
explicitly adopt and follow coding style guides in their work.
Style guides harmonize code within and across teams 
making it easier to understand and reuse code,
which ultimately helps teams to 
build on each other's best practices.
Some of the programming languages used at DIME 
already have well-established and commonly used style guides,
such as the Tidyverse style guide for R
and PEP-8 for Python.\sidenote{See DIME Analytics Coding Standards:
	\url{https://github.com/worldbank/dime-standards}}. 
```

Stata has relatively few resources of this type available,

> which is why we have created and shared one here that

Reviewer: ```suggestion
which is why we have created and included one here that
```
